data/data/rhcos-amd64.json

```diff
@@ -1,135 +1,135 @@
         },
         "us-east-1": {
-            "hvm": "ami-031e3849b97db693c"
+            "hvm": "ami-04893bec8e83d1cb0"
         },
         "us-east-2": {
-            "hvm": "ami-0af3834ffb0820d02"
+            "hvm": "ami-0e888b699fa6e37e7"
         },
         "us-west-1": {
-            "hvm": "ami-07e8274bc6609864b"
+            "hvm": "ami-053bc51af84d379d4"
         },
         "us-west-2": {
-            "hvm": "ami-0f0a99d2f300aa658"
+            "hvm": "ami-0e612d41d145987d9"
         }
     },
     "azure": {
-        "image": "rhcos-44.81.202002241126-0-azure.x86_64.vhd",
-        "url": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-44.81.202002241126-0-azure.x86_64.vhd"
+        "image": "rhcos-44.81.202003062006-0-azure.x86_64.vhd",
+        "url": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-44.81.202003062006-0-azure.x86_64.vhd"
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.4/44.81.202002241126-0/x86_64/",
-    "buildid": "44.81.202002241126-0",
+    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.4/44.81.202003062006-0/x86_64/",
+    "buildid": "44.81.202003062006-0",
     "gcp": {
-        "image": "rhcos-44-81-202002241126-0-gcp-x86-64",
-        "url": "https://storage.googleapis.com/rhcos/rhcos/rhcos-44-81-202002241126-0-gcp-x86-64.tar.gz"
+        "image": "rhcos-44-81-202003062006-0-gcp-x86-64",
+        "url": "https://storage.googleapis.com/rhcos/rhcos/rhcos-44-81-202003062006-0-gcp-x86-64.tar.gz"
     },
     "images": {
         "aws": {
-            "path": "rhcos-44.81.202002241126-0-aws.x86_64.vmdk.gz",
-            "sha256": "2a6a1c5ba0dd6819c1df11e210e1752dead6e92bfc8dc9f88ff4bdfe826589af",
-            "size": 866196706,
-            "uncompressed-sha256": "e480f3cd790f81c94f166b8706ee7aba51ef474c6a843b36e54c597563d19436",
-            "uncompressed-size": 884048896
+            "path": "rhcos-44.81.202003062006-0-aws.x86_64.vmdk.gz",
+            "sha256": "3c96142f5f9c2362864abb362f99256eb6e3b5ca01740da39e7ad251f2a42ff6",
+            "size": 862543887,
+            "uncompressed-sha256": "f5a93a7616835785a74d54c7ef123a2ef634124e6f6c4b743b310e5f6b2c5404",
+            "uncompressed-size": 880251904
         },
         "azure": {
-            "path": "rhcos-44.81.202002241126-0-azure.x86_64.vhd.gz",
-            "sha256": "d7b3fcc3cce5c27bd5b0b844d014efdf2f8a1979b48c24d8c701b2fb4fd16a36",
-            "size": 866305735,
-            "uncompressed-sha256": "83e42fe90c1e6a1d742c995dc0354e41d739c5bb4c488f8f621962570a6fb2af",
+            "path": "rhcos-44.81.202003062006-0-azure.x86_64.vhd.gz",
+            "sha256": "45f5838fea89db9757be9a02e68515c6dfc9e8ebaf39f65dc90e1219261ede8f",
+            "size": 862671452,
+            "uncompressed-sha256": "8b6cb9a49697c1a1ecc1a121f4d4b1add397cc4f7c1ea67d49de6384b7e07da5",
             "uncompressed-size": 17179869696
```

Comment:
hmm.. the uncompressed size hasn't changed :P

Reply:
THAT does look odd.

Reply:
Looks like that number hasn't change since bd6291c17a1ff1fc9f4a378efe34b30eb810beef. I'll open a bug up against the build pipeline to ensure that it starts getting updated once more.

Reply:
```
$ ls -l rhcos-44.81.202003062006-0-azure.x86_64.vhd
-rw-rw-r--+ 1 steve steve 17179869696 Mar  6 20:07 rhcos-44.81.202003062006-0-azure.x86_64.vhd
```

Reply:
It looks like in cosa we're using:

```
    "azure": {
        "image_format": "vpc",
        "image_suffix": "vhd",
        "platform": "azure",
        "convert_options": {
            '-o': 'force_size,subformat=fixed'
        },
    },
```

so...hm, the `force_size` isn't documented in `man qemu-img` but a quick glance at the qemu source shows:

```

    /*
     * Calculate matching total_size and geometry. Increase the number of
     * sectors requested until we get enough (or fail). This ensures that
     * qemu-img convert doesn't truncate images, but rather rounds up.
     *
     * If the image size can't be represented by a spec conformant CHS geometry,
     * we set the geometry to 65535 x 16 x 255 (CxHxS) sectors and use
     * the image size from the VHD footer to calculate total_sectors.
     */
    if (vpc_opts->force_size) {
        /* This will force the use of total_size for sector count, below */
        cyls         = VHD_CHS_MAX_C;
        heads        = VHD_CHS_MAX_H;
        secs_per_cyl = VHD_CHS_MAX_S;
    } else {
        total_sectors = MIN(VHD_MAX_GEOMETRY, total_size / BDRV_SECTOR_SIZE);
        for (i = 0; total_sectors > (int64_t)cyls * heads * secs_per_cyl; i++) {
            calculate_geometry(total_sectors + i, &cyls, &heads, &secs_per_cyl);
        }
    }
```

Looks like rounding to sector size.  If the contents haven't changed enough to allocate a new sector, then the uncompressed size will be the same.

So I think this one is fine too.